### PR TITLE
R4R: bep3 module upgrades

### PR DESCRIPTION
**Pull request by denalimarsh**
This PR updates the bep3 module to be compatible with the bep3-deputy process:
- Timestamp param "now" defaults to `time.Now().Unix` unless it is manually set i.e. `1583452892` (deputy-created swaps will have manually set timestamps)
- Secure number generation via `crypto/rand` with proper hexadecimal encoding. Print statements updated to match bnbchain
- Update asset denom from `btc` to `bnb`, moved genesis files to `/testnet-5000`

Commit history is messy from merge, reference 'Files Changed' instead.

### CODE REVIEW

File: x/bep3/client/cli/tx.go [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
 import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
 	"fmt"
+	"math/big"
 	"strconv"
 	"strings"
+	"time"
```
Comment (karzak) [inline x/bep3/client/cli/tx.go]:
```suggestion
	tmtime "github.com/tendermint/tendermint/types/time"
```

File: x/bep3/client/cli/tx.go [hunk 2]
```diff
@@ -54,36 +59,50 @@ func GetCmdCreateAtomicSwap(cdc *codec.Codec) *cobra.Command {
 			recipientOtherChain := args[1] // same as OtherExecutor.DeputyAddress
 			senderOtherChain := args[2]
 
-			randomNumberHash, err := types.HexToBytes(args[3])
-			if err != nil {
-				return err
+			// Timestamp defaults to time.Now() unless it's explicitly set
+			var timestamp int64
+			if strings.Compare(args[3], "now") == 0 {
+				timestamp = time.Now().Unix()
```
Comment (karzak) [inline x/bep3/client/cli/tx.go]:
```suggestion
				timestamp = tmtime.Now().Unix()
```

Use tendermint's Now method for time.Now

File: x/bep3/keeper/swap.go [hunk 3]
```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/hex"
 	"fmt"
+	"time"
```
Comment (karzak) [inline x/bep3/keeper/swap.go]: See previous comment

Comment (denalimarsh) [reply x/bep3/keeper/swap.go]: https://github.com/Kava-Labs/kava/pull/388/commits/5c4832e2209b183469c84c9b3fa5218114da6b75

File: x/bep3/client/cli/tx.go (hunk 2 above)
Comment (denalimarsh) [reply x/bep3/client/cli/tx.go]: https://github.com/Kava-Labs/kava/pull/388/commits/5c4832e2209b183469c84c9b3fa5218114da6b75

File: x/bep3/client/cli/tx.go (hunk 1 above)
Comment (denalimarsh) [reply x/bep3/client/cli/tx.go]: https://github.com/Kava-Labs/kava/pull/388/commits/5c4832e2209b183469c84c9b3fa5218114da6b75

**Pull request merged**